pypeit/wavetilts.py

```diff
@@ -70,7 +70,7 @@ def __init__(self, msarc, tslits_dict, spectrograph, par, wavepar, det=1,
         self.det = det
         self.redux_path = redux_path
         #
-        if self.spectrograph.detector is not None:
+        if self.spectrograph is not None:
```

Comment:
isn't `spectrgraph` required?  It should be.
Oh, maybe it needs to be `None` on occasion..

Reply:
It is currently required.